src/include/ZividPython/Traits.h

```diff
@@ -0,0 +1,45 @@
+#pragma once
+
+#include <type_traits>
+
+#if __has_include(<experimental/type_traits>)
+#    include <experimental/type_traits>
+#endif
+
+namespace
```

Comment:
```suggestion
namespace Detail
```

https://wiki.sei.cmu.edu/confluence/display/cplusplus/DCL59-CPP.+Do+not+define+an+unnamed+namespace+in+a+header+file

Reply:
There is a clang-tidy check for this, consider enabling it, if it is not already enabled.

Reply:
https://clang.llvm.org/extra/clang-tidy/checks/google-build-namespaces.html

Reply:
Ah yes, I knew that. Forgot I was in a header...

I realize we don't run clang-tidy at all in this repo yet. Enabling that seems to me as too much a scope-creep for a bugfix PR. Do you agree?

Reply:
Anyway, put it in a Detail namespace now.

Reply:
We should enable it then, but that is a task in itself.